docs/extensions/filters/writing_custom_filters.md

```diff
@@ -67,9 +67,10 @@ To extend Quilkin's code with our own custom filter, we need to do the following
    // src/main.rs
    use quilkin::filters::{Filter, ReadContext, ReadResponse, WriteContext, WriteResponse};
 
+   const NAME: &str = "greet.v1";
```

Comment:
I was hoping that #319 's snapshotting would solve this issue, but looks like that's blocked for the moment.

Since this is a change in API, and this will break the docs for 0.1.0 -- I'm wondering how to handle this in the short term. 

Maybe we add a note here: 
![image](https://user-images.githubusercontent.com/298370/126222883-fc3f113e-d59d-4b16-bc87-56a447dbc3b4.png)

That says something like:

> This documentation is for the development version of Quilkin. To view the documentation for a specific release, switch to the release tag in the above dropdown.

(I'm happy to do this as a seperate PR if we are all happy with it).

Which can then we can remove it once #319 is merged and good to go.

Other than that, this is 👍🏻  as far as I am concerned - and I want to get it in, as I know a lot of your follow up work is based on this.

Sound good?

Reply:
SGTM, I can make a separate PR.